specs/eip4844/p2p-interface.md

```diff
@@ -140,6 +141,7 @@ Per `context = compute_fork_digest(fork_version, genesis_validators_root)`:
 | `GENESIS_FORK_VERSION`   | `phase0.SignedBeaconBlock` |
 | `ALTAIR_FORK_VERSION`    | `altair.SignedBeaconBlock` |
 | `BELLATRIX_FORK_VERSION` | `bellatrix.SignedBeaconBlock` |
+| `CAPELLA_FORK_VERSION`   | `capella.SignedBeaconBlock`   |
```

Comment:
Potential merge conflict with #3089. I can fix that after this is merged

Reply:
fixed in https://github.com/ethereum/consensus-specs/pull/3052/commits/f1d4c9047a70c3a321cb0329216d446ead17e007